journals/apps/journals/models.py

```diff
@@ -314,6 +315,17 @@ def generate_basket_url(self, sku):
         ))
         return basket_url
 
+    def update_related_objects(self, deactivate=False):
+        data = {
+            "status": "active" if not deactivate else "inactive",
+            "card_image_url": self.site.root_url + self.card_image.file.url,
```

Comment:
how, if any, will this change when we use S3 backend for assests?

Reply:
Hmmm, good point. We definitely won't want to build the URL this way. I'll see if I can find a way to get a FQDN URL.